tests/fixtures/mod.rs

```diff
@@ -105,6 +105,8 @@ where
         .arg("--rm")
         .arg("--label")
         .arg("testcontainer")
+        .arg("--network")
+        .arg("wiresmith")
```

Comment:
This would have all of the tests share the same network which means that running tests in parallel might lead to weird behavior right?

Reply:
Each network is named after the pattern `wiresmith-{http_port}` now.